src/integrations/lark/types/recording-file.types.ts

```diff
@@ -12,7 +12,7 @@
 export interface RecordingFileData {
   record_file_id: string;
   meet: string[];
-  participants?: string;
+  participants?: string[];
```

Comment:
为什么需要调整  participants?: string为participants?: string[];

```suggestion
  participants?: string;
```